static/app/views/dashboards/addWidget.tsx

```diff
@@ -160,7 +160,14 @@ export function AddWidgetButton({onAddWidget, ...buttonProps}: Props & ButtonPro
         key: DataSet.METRICS,
         label: t('Metrics'),
         onAction: () => handleAction(DataSet.METRICS),
-        trailingItems: <FeatureBadge type="beta" />,
+        trailingItems: (
+          <FeatureBadge
```

Comment:
@ale-cota I am wondering if we should replace the feature badge with the deprecated badge

Reply:
@priscilawebdev thinking about it now, since it was just a beta and never launched, it maybe doesn't make sense to start adding deprecated in some places. So let's just leave the beta badge with the message in the tooltip as you added. Thank you 🙏